### PR TITLE
bugfix - installers and Dockerfile compatible with arm64

### LINKED ISSUE #6143: new installers with checksum not compatible with arm64
If you are trying to resolve an environment-specific issue or have a one-off question about the edge case that does not require a feature then please consider asking a question in argocd slack [channel](https://argoproj.github.io/community/join-slack).

Checklist:

* [x] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**

When building argocd with rpi4, it breaks during the installer, as the checksum files for arm64 are not included

**To Reproduce**

Build with arm64 architecture

**Expected behavior**

build without issue

**Version**

latest master code

**Logs**

```
build-argocd-kt74x-3206545311: + grep -q helm-v2.17.0-linux-arm64.tar.gz /tmp/installers/checksums/helm-v2.17.0-linux-arm64.tar.g
z.sha256
build-argocd-kt74x-3206545311: grep: /tmp/installers/checksums/helm-v2.17.0-linux-arm64.tar.gz.sha256: No such file or directory
build-argocd-kt74x-3206545311: + echo No checksum for helm-v2.17.0-linux-arm64.tar.gz in /tmp/installers/checksums/helm-v2.17.0-l
inux-arm64.tar.gz.sha256
build-argocd-kt74x-3206545311: No checksum for helm-v2.17.0-linux-arm64.tar.gz in /tmp/installers/checksums/helm-v2.17.0-linux-ar
m64.tar.gz.sha256
build-argocd-kt74x-3206545311: + exit 1
build-argocd-kt74x-3206545311: v1.21.9error building image: error building stage: failed to execute command: waiting for process
to exit: exit status 1
```

### PR DESCRIPTION
**Pull request by mickkael**
Closes #6143

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,7 +9,6 @@ FROM docker.io/library/golang:1.16.2 as builder
 RUN echo 'deb http://deb.debian.org/debian buster-backports main' >> /etc/apt/sources.list
 
 RUN apt-get update && apt-get install -y \
-    openssh-server \
```
Comment (alexmt) [inline Dockerfile:12]: The openssh-server is required for e2e tests. Does it have to be removed ?

File: Dockerfile [hunk 2]
```diff
@@ -29,7 +28,6 @@ ADD hack/installers installers
 ADD hack/tool-versions.sh .
 
 RUN ./install.sh packr-linux
-RUN ./install.sh ksonnet-linux
```
Comment (alexmt) [inline Dockerfile:32]: Unfortunately, we cannot stop support ksonnet yet. Is it possible to conditionally skip it for arm?

**Review by alexmt — CHANGES_REQUESTED** [review]
Thanks for working on fixing ARM support! Added two comments. Let me know what you think please.

File: Dockerfile (hunk 1 above)
Comment (mickkael) [reply Dockerfile]: reverted openssh-server removal. @alexmt

File: Dockerfile (hunk 2 above)
Comment (mickkael) [reply Dockerfile]: I have reverted the ksonnet install removal, and fix the arm64 install instead. Please check. @alexmt

**Comment by mickkael** [comment]
made a mistake with the sign-off, will re-submit

**Pull request closed without merging**